### ISSUE TITLE
Make the UI touch-friendly

### ISSUE THREAD

**Issue by MarcAnt01** [bigbluebutton/bigbluebutton#10594]
**Is your feature request related to a problem? Please describe.**
BBB works with touchscreen, but menus are a bit to small to be easily used with touchscreen

**Describe the solution you'd like**
Make text a bit bigger and increase padding

**Describe alternatives you've considered**
None

**Additional context**
Sites like Google Meet, Google Photos, YouTube have UIs already optimized for touch usage, you could take a look at them

**Comment by matiasilva** [comment]
Can you provide any further detail to your feature request, please?

**Comment by MarcAnt01** [comment]
@matiasilva I have tried to be clear, but I assume I haven't managed, can you tell me what you haven't understood?

**Comment by matiasilva** [comment]
Is the request to just make the text bigger and increase padding on mobile?

**Comment by MarcAnt01** [comment]
Yes, but not on mobile only.
I use a 13.5 inches Surface Book 2 that I sometimes use as a tablet and some menus like the ones with options on a video conference and the settings are a bit to small. Increasing the font size will make the ui too big

**Comment by MarcAnt01** [comment]
![image](https://user-images.githubusercontent.com/43090263/95251265-97de4e80-081b-11eb-94ac-213745b4629f.png)
An example of touch friendly UI is this menu from WinUI

**Comment by MarcAnt01** [comment]
@matiasilva I tried BBB with a resized window and it is touch friendly, currently the issue is to make it touch friendly also with an expanded window. It is not a matter of screen size, but of finger size, if the menus are too close to each other you risk to click the wrong item

**Comment by matiasilva** [comment]
As this is a very visual suggestion, could you provide exact screenshots of what you want to be improved? Can you mock a few things up using the developer tools?

**Comment by MarcAnt01** [comment]
Parts that need to be made touch-friendly
![image](https://user-images.githubusercontent.com/43090263/96467128-e849a900-122a-11eb-8bae-32a20db26b6e.png)
![image](https://user-images.githubusercontent.com/43090263/96467185-f1d31100-122a-11eb-8173-0adcd590ccfb.png)
![image](https://user-images.githubusercontent.com/43090263/96467225-fd263c80-122a-11eb-82dd-85dbb57fa024.png)
![image](https://user-images.githubusercontent.com/43090263/96467270-08796800-122b-11eb-9f3d-40b701e4a6d4.png)

**Comment by matiasilva** [comment]
So, you propose increasing the padding between the menu items?

**Comment by MarcAnt01** [comment]
@matiasilva yes, but also slightly increase the font there, otherwise the menu will look a bit weird.
